README.rst

```diff
@@ -10,10 +10,25 @@ pagination, and more.
 This is relatively stable. We're using it in production and have attempted
 to document the important parts, but feedback is welcome.
 
+Breaking change in 0.7.0
```

Comment:
What do you think about calling this 1.0.0? I know we haven't strictly been using semantic versioning here, but maybe this is a good time to start.

Reply:
I considered calling it 1.0.0, so let's do it!